githooks/pre-push.sh

```diff
@@ -100,14 +100,14 @@ do
 		try_run ./cheribuild.py -p __run_everything__ --clean --build --test --benchmark
 		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --allow-more-than-one-target --build --test --cpu=default -p __run_everything__
 		# Regression for --benchmark-clean-boot:
-		try_run ./cheribuild.py mibench-mips64 --benchmark --benchmark-clean-boot -p
-		try_run ./cheribuild.py mibench-mips64-purecap --benchmark --benchmark-clean-boot -p
+		# TODO: try_run ./cheribuild.py mibench-new-riscv64 --benchmark --benchmark-clean-boot -p
+		# TODO: try_run ./cheribuild.py mibench-new-riscv64-purecap --benchmark --benchmark-clean-boot -p
 		# Various jenkins things that have failed in the past
 		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --build --test --tarball -p libcxx-riscv64-purecap
-		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --build --cpu=cheri128 -p run-mips64-purecap
+		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --build --cpu=cheri128 -p run-morello-purecap
```

Comment:
--cpu doesn't really make sense any more

Reply:
I agree, and it hasn't for a long time. Planning to remove that in a separate change though